src/main/java/hudson/plugins/accurev/delegates/StreamDelegate.java

```diff
@@ -88,6 +91,31 @@ protected PollingResult checkForChanges(Job<?, ?> project)
           .println("Tried to find '" + localStream + "' Stream, could not found it.");
       return PollingResult.NO_CHANGES;
     }
+    // command to get version of accurev
+    final ArgumentListBuilder cmd = new ArgumentListBuilder();
+    cmd.add("accurev");
```

Comment:
```suggestion
```

Reply:
if (!accurevPath.equals(machineReadableCommand.toCommandArray()[0])) {
      machineReadableCommand.prepend(accurevPath);
    }
this line launcher run command may give nullpointer na?